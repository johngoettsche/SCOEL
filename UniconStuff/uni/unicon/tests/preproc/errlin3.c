/* test error message: bad file name on line directive */

#line 1 +
