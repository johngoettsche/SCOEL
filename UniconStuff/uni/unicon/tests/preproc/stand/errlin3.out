pp: File errlin3.c; Line 3: syntax error in #line
